src/app/RosControlTest/CMakeLists.txt

```diff
@@ -4,4 +4,6 @@ target_sources(RosControlTest PRIVATE main.cpp)
 
 target_link_libraries(RosControlTest PRIVATE hardware ros-handler)
 
-MakeROSTarget(RosControlTest)
\ No newline at end of file
+MakeROSTarget(RosControlTest)
+
+MakeTalonTarget(RosControlTest)
```

Comment:
This file, once the running code is moved, probably does not need to be a Talon target.